Update ACT setup guide (#3)
OverlayPlugin/docs: setup/README.md

![Installing the Parsing Plugin in ACT](resources/parsing_plugin.png)

Click **Next** to move to the log file section. ACT will ask if it will be used for Final Fantasy XIV. Select **Yes** to configure ACT logs for FFXIV.

![Log File Screen in the Wizard](resources/log_file.png)

Click **Next** to move to Startup Settings, then **Close** to accept the default settings and finish the startup wizard.

At this point `FFXIV_ACT_Plugin.dll` should be enabled in **Plugins** > **Plugin Listing**.

![Plugin Listing Tab](resources/ffxiv_act_plugin.png)

Next, you will need to decide whether you wish to use the older packet capture (PCAP) to gather the game's network traffic,
or the newer Deucalion injection. Deucalion is not officially recommended as the default option yet, but does not require
ACT to be started prior to logging in to your character, does not require the firewall exception, and does not require
running as admin. On the other hand, it is potentially more dangerous should SE decide to take action against 
injection-based plugins, and has not been as thoroughly tested as plain packet capture.

### PCAP Method

#### Running as Admin

> It is recommended that ACT be run as Admin. You can right-click on the ACT shortcut and select the **Run as administrator** option. You may get an UAC prompt, select yes.

Reviewer: This doesn't have to be done in this PR (and this text was here before), but I wonder if we should not recommend to run ACT as admin anymore.

Author: IIRC default launcher still elevates after a patch?

Reviewer: Mmm, that's fair.  It still seems like bad practice to me to recommend people run it as admin, but maybe that's an ok compromise here.

`Advanced Combat Tracker` should now appear in the list of **Allowed apps and features**. Click **OK** and exit the Control Panel.

![Allowed Apps List](resources/allowed_apps_act.png)

### Injection Method

All you need to do to make use of injection is check this box:

![Deucalion/Injection Checkbox](resources/deucalion_checkbox.png)

## OverlayPlugin

From the **Plugin Listing** tab, click on the `Get Plugins...` button near the upper right corner. This will open a window that will populate with available plugins for ACT.

![Get Plugins Button](resources/get_plugins.png)

In the **Get Plugins** window, select the `Overlay Plugin` option and click on `Download and Enable`. This will add the latest **OverlayPlugin** to ACT (the OverlayPlugin auto-updater may also run during this step). 

![Get Plugins Window](resources/get_plugins_window.png)

The OverlayPlugin should now be setup. Click on the `X` to close the **Get Plugins** window.

**At this point it is recommended to restart ACT before continuing on.**

### Using OverlayPlugin to End Encounters

It is recommended to use OverlayPlugin's in/out-of-combat detection to split encounters, rather than ACT's less accurate behavior.

> To do this, first find ACT's encounter split timeout (Options > Main Table/Encounters > General). Either disable it entirely by un-checking the box, or set it to a relatively large value:

Reviewer: The text "set it to a relatively large value" plus a screenshot that suggests "6" is very misleading.  I continue to think it's better to give the same advice that's in discord pins for consistency.  There have been a lot of questions around this over time.

```suggestion
To do this, first find ACT's encounter split timeout (Options > Main Table/Encounters > General).
It is recommended to disable it entirely by un-checking both `Number of seconds to wait` checkboxes:
```

![get_plugins](resources/get_plugins.png)

> In the **Get Plugins** window, select the `[FFXIV+others]Overlay Plugin` option and click on `Download and Enable`. This will add the latest **OverlayPlugin** to ACT (the OverlayPlugin auto-updater may also run during this step). 

Reviewer 2: it no longer has [FFXIV+others] in front of the plugin in the list

Reviewer: This looks like it's been resolved, so for the sake of getting this landed I'm just going to merge anyway with this outstanding "change request".

> In the **Get Plugins** window, select the `[FFXIV+others]Overlay Plugin` option and click on `Download and Enable`. This will add the latest **OverlayPlugin** to ACT (the OverlayPlugin auto-updater may also run during this step). Note that the number displayed next to the plugin may differ from the one shown here.

Reviewer: Would you mind updating this screenshot to reflect the changed id?

Author: Done

![get_plugins_window](resources/get_plugins_window.png)

The OverlayPlugin should now be setup. Click on the `X` to close the **Get Plugins** window.

**At this point it is recommended to restart ACT before continuing on.**

### Using OverlayPlugin to End Encounters

It is recommended to use OverlayPlugin's in/out-of-combat detection to split encounters, rather than ACT's less accurate behavior.

> To do this, first set ACT's encounter split timeout (Options > Main Table/Encounters > General) to a large value, such as 60 seconds (or even higher, for duties with long cutscenes):

Reviewer: Would it be worth recommending just unclicking this box entirely?

Author: Hmm, I don't know. I think it's still worth having some kind of value there for fresh patches when OP's memory sigs might not be updated yet.

Reviewer: I sorta disagree.  Most of the time it'll break on wipes which come from ACT log lines, which is good enough for most use cases?  I've had it unchecked for a very long time with no issues.

Reviewer: Mostly I also mention this because this is also Aho's advice in #general pins: https://cdn.discordapp.com/attachments/551474815727304706/921609700842299413/Step1.PNG

So, I figure it makes sense for the setup guide to match the pins? Or we change both of them?

Author: I guess I can just mention both ways of doing it

Reviewer: You're saying that you disagree that the setup guide and the pins should have the same recommendation?

Author: Well, I didn't write the pin, so I can't vouch for the information.

There are a few situations where you're still "in combat" but it shouldn't be considered the same pull, e.g. Praetorium when people tag mobs and then hit the teleporter (unsure if it happens with the remade version). The ACT timer is more of a failsafe for those situations. 

Also, more generally, I think it's fine for a pin to be the short version, and then the guide can elaborate on it further.

Reviewer: Belatedly, I don't think there's any vouching required here.  The pin says:

```
About : Handling ACT Encounters

Here's the recommendation : 
First, turn off that option in ACT Settings. 
>Then, you can either let OverlayPlugin handle it
>Or Triggernometry (Misc repo)
>Or you can make an in-game macro (but you'll have to press it everytime)
```
![image](https://user-images.githubusercontent.com/462317/229844695-806156ad-37eb-403c-81ab-b605f7fb2eb4.png)
(plus some other images about how to have trigg or overplayplugin handle it)

Belatedly, I remember that 60 seconds is not enough for some fight cutscenes, which is why I also think "don't click the checkbox" is a better recommendation.  I also continue to think that the pin in the discord and this guide should have the same advice to be less confusing to users.

> ![act timeout settings](https://user-images.githubusercontent.com/14287379/199086818-7509e165-70a8-49bc-b43c-2ccfa3a9235b.png)

Reviewer: Can you check these images into the repo?

Author: Done
